### ISSUE TITLE
Can't import the file

### ISSUE THREAD

**Issue by chinghsi** [SOLAR-group/TAWOS#3]
I can't import the sql file to mysql database successfully. It seems like the data contains single quotes and they mix with mysql scripts.

**Comment by vtawosi** [comment]
Hi @chinghsi! 
As the sql file is exported via mysql, all the quotes (single and double) are skipped properly and it should not be a problem. Can you please confirm that the file is not opened/edited by other apps (like text editors) after being downloaded and before import?

**Comment by chinghsi** [comment]
@vtawosi Thank you for the reply. 
I redownload the file and try to import it by running` mysql -u root -p tawos < TAWOS.sql`. Then it returns the error message: `ERROR at line 159: Unknown command '\"'.`

**Comment by vtawosi** [comment]
Hi @chinghsi. Unfortunately, I was unable to reproduce your error. Would you please be able to install MySQL Workbench (https://www.mysql.com/products/workbench) and try importing the TAWOS.sql from within the application? This might resolve the issue, or provide you with a log that may help find the true cause of the problem.

**Comment by chinghsi** [comment]
Hi @vtawosi. Sorry for the late reply. I import the sql file successfully with Workbench. Thank you very much.

Here was my experience:
I opened TAWOS.sql and it showed "Assertion failed" message. But ignoring the message and running the script, it worked.
![image](https://user-images.githubusercontent.com/57378681/230550137-284194d1-e057-48e4-aab7-af408f69e8b2.png)

I also tried editing TAWOS.sql to separate each table to single sql file. Each file didn't show the above message dialog and could import successfully.
